databricks/koala/structures.py

```diff
@@ -588,6 +588,24 @@ def pipe(self, func, *args, **kwargs):
     def shape(self):
         return len(self), len(self.columns)
 
+    def sample(self, n=None, frac=None, replace=False, random_state=None):
```

Comment:
I think it's better to add the same parameters as Pandas. And, we raise a `NotImplementedError`.
```python
def(self, n=None, frac=None, replace=False, weights=None, random_state=None, axis=None)
```